src/main/java/org/jabref/gui/actions/CopyVersionToClipboardAction.java

```diff
@@ -1,23 +0,0 @@
-package org.jabref.gui.actions;
-
-import java.awt.event.ActionEvent;
-
-import javax.swing.AbstractAction;
-
-import org.jabref.Globals;
-import org.jabref.JabRefGUI;
-import org.jabref.gui.ClipBoardManager;
-import org.jabref.logic.l10n.Localization;
-import org.jabref.logic.util.BuildInfo;
-
-public class CopyVersionToClipboardAction extends AbstractAction {
```

Comment:
Functionality totally removed?

Reply:
There is still a "copy version" button in the about dialog but no separate menu entry. (I didn't change anything as part of this PR; this class was completely unused).